index.js

```diff
@@ -3,18 +3,19 @@
 const git = require('./lib/git');
 
 const services = {
-  travis: require('./lib/travis'),
```

Comment:
Had a merge conflict here so I sorted this, too.

Reply:
👍